src/nft/components/explore/Table.tsx

```diff
@@ -186,9 +191,11 @@ export function Table<D extends Record<string, unknown>>({
                   <td className={clsx(styles.td, classNames?.td)} {...cell.getCellProps()} key={cellIndex}>
                     {cellIndex === 0 ? (
                       <RankCellContainer>
-                        <ThemedText.BodySecondary fontSize="14px" lineHeight="20px">
-                          {i + 1}
-                        </ThemedText.BodySecondary>
+                        {!isMobile && (
```

Comment:
nit: I think it'd be easier in the future to handle this within css. Display none if it's below a breakpoint. That way we can add further conditional breakpoints without needing to add more hooks to the component, and it separates the concerns a bit more. Not blocking though.